examples/lines_bars_and_markers/angle_marker.py

```diff
@@ -0,0 +1,323 @@
+line2, = ax1.plot(*zip(*p2))
+point, = ax1.plot(*center, marker="o")
+
+am1 = AngleMarker(center, p1[1], p2[1], ax=ax1, size=75, text=r"$\alpha$")
+am2 = AngleMarker(center, p2[1], p1[0], ax=ax1, size=35, text=r"$\beta$")
+am3 = AngleMarker(center, p1[0], p2[0], ax=ax1, size=75, text=r"$\gamma$")
+am4 = AngleMarker(center, p2[0], p1[1], ax=ax1, size=35, text=r"$\theta$")
+
+
+# Showcase some styling options for the angle arc, as well as the text
+p = [(6.0, 400), (5.3, 410), (5.6, 300)]
+ax1.plot(*zip(*p))
+am5 = AngleMarker(p[1], p[0], p[2], ax=ax1, size=40, text=r"$\Phi$",
+                  linestyle="--", color="gray", textposition="outside",
+                  text_kw=dict(fontsize=16, color="gray"))
+
+
+#### SUBPLOT 2 ####
+# Helper function to draw angle easily
+def plot_angle(ax, pos, angle, length=0.95, acol="C0", **kwargs):
+    vec2 = np.array([np.cos(np.deg2rad(angle)), np.sin(np.deg2rad(angle))])
+    xy = np.c_[[length, 0], [0, 0], vec2*length].T + np.array(pos)
+    ax.plot(*xy.T, color=acol)
+    return AngleMarker(pos, xy[0], xy[2], ax=ax, **kwargs)
+
+# Showcase different textpositions
+kw = dict(size=75, unit="points", text=r"$60°$")
+
+am6 = plot_angle(ax2, (2.0, 0), 60, textposition="inside", **kw)
+am7 = plot_angle(ax2, (3.5, 0), 60, textposition="outside", **kw)
+am8 = plot_angle(ax2, (5.0, 0), 60, textposition="edge",
+                 text_kw=dict(bbox=dict(boxstyle="round", fc="w")), **kw)
+am9 = plot_angle(ax2, (6.5, 0), 60, textposition="edge",
+                 text_kw=dict(xytext=(30, 20), arrowprops=dict(arrowstyle="->",
+                              connectionstyle="arc3,rad=-0.2")), **kw)
+
+ax2.annotate("textpostion", xy=(.02, .9), xycoords="axes fraction",
+                 bbox=dict(boxstyle="round", fc="w"), ha="left")
+for x, text in zip([2.0, 3.5, 5.0, 6.5], ['"inside"', '"outside"', '"edge"',
+                                          '"edge", custom arrow']):
+    ax2.annotate(text, xy=(x, 0), xycoords=ax2.get_xaxis_transform(),
+                 bbox=dict(boxstyle="round", fc="w"), ha="left", fontsize=8)
+
+#### SUBPLOT 3 ####
+# Showcase different size units. The effect of this can best be observed
+# by interactively changing the figure size
+kw = dict(text=r"$60°$", textposition="outside")
+
+am10 = plot_angle(ax3, (2.0, 0), 60, size=50, unit="pixels", **kw)
+am11 = plot_angle(ax3, (3.5, 0), 60, size=50, unit="points", **kw)
+am12 = plot_angle(ax3, (5.0, 0), 60, size=0.25, unit="axes min", **kw)
+am13 = plot_angle(ax3, (6.5, 0), 60, size=0.25, unit="axes max", **kw)
+
+ax3.annotate("unit", xy=(.02, .9), xycoords="axes fraction",
+                 bbox=dict(boxstyle="round", fc="w"), ha="left")
+for x, text in zip([2.0, 3.5, 5.0, 6.5], ['"pixels"', '"points"',
+                                          '"axes min"', '"axes max"']):
+    ax3.annotate(text, xy=(x, 0), xycoords=ax3.get_xaxis_transform(),
+                 bbox=dict(boxstyle="round", fc="w"), ha="left", fontsize=8)
+
+fig.tight_layout()
```

Comment:
This is not applied:
~~~
/home/circleci/project/examples/lines_bars_and_markers/angle_marker.py:303: UserWarning: Tight layout not applied. The left and right margins cannot be made large enough to accommodate all axes decorations.
  fig.tight_layout()
~~~

Either find out what to do about it, or just remove it. The plot looks good also without that.